changelog/379.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Attributes of _WARNING_DETAILS are checked to make sure they can be dumped
```

Comment:
`_WARNING_DETAILS` is too much of an implementation detail which makes little sense to general users. I suggest:

```
Warning attributes are checked to make sure they can be dumped prior to serializing the warning for submission to the master node.
```